content/en/docs/contributing/docs/style-guide.md

```diff
@@ -0,0 +1,400 @@
+Use meaningful variable names that have a context. | Use variable names such as 'foo','bar', and 'baz' that are not meaningful and lack context.
+Remove trailing spaces in the code. | Add trailing spaces in the code, where these are important, because the screen reader will read out the spaces as well.
+
+### Use code style for object field names
+
+Do | Don't
+:--| :-----
+Set the value of the `interval` field in the configuration file. | Set the value of the "interval" field in the configuration file.
+The value of the `chart` field is an HelmChartTemplate object. | The value of the "chart" field is an HelmChartTemplate object.
+
+### Use code style for fluxcd command tool and component names
+
+Do | Don't
+:--| :-----
+The Source controller preserves node stability. | The `source-controller` preserves node stability.
+The `flux` tool handles installation. | The flux tool handles installation.
+Run the command to get Kustomizations with the name, `flux get kustomization <name>`. | Run the command to get Kustomizations with the name, flux get kustomization \<name\>. |
+
+### Starting a sentence with a component tool or component name
+
+Do | Don't
+:--| :-----
+The `flux` tool bootstraps and provisions Flux controllers in a cluster. | `flux` tool bootstraps and provisions Flux controllers in a cluster.
+The Source controller manages resources. | Source controller manages resources.
+
+### Use a general descriptor over a component name
+
+Do | Don't
+:--| :-----
+The Flux Source controller offers an OpenAPI spec. | The source-controller offers an OpenAPI spec.
+
+### Use normal style for string and integer field values
+
+For field values of type string or integer, use normal style without quotation marks.
+
+Do | Don't
+:--| :-----
+Set the value of `imagePullPolicy` to Always. | Set the value of `imagePullPolicy` to "Always".
+Set the value of `image` to nginx:1.16. | Set the value of `image` to `nginx:1.16`.
+Set the value of the `replicas` field to 2. | Set the value of the `replicas` field to `2`.
+
+## Code Snippet Formatting
+
+### Don't include the command prompt
+
+Do | Don't
+:--| :-----
+flux get kustomizations | $ flux get kustomizations
+
+### Separate commands from output
+
+Verify that your cluster satisfies the prerequisites with:
+
+```shell
+flux check --pre
+```
+
+The output is similar to this:
+
+```console
+► checking prerequisites
+✔ kubectl 1.18.3 >=1.18.0
+✔ kubernetes 1.18.2 >=1.16.0
+✔ prerequisites checks passed
+```
+
+## Fluxcd.io Word List
+
+A list of Flux-specific terms and words to be used consistently across the site.
+
+Term | Usage
+:--- | :----
+Flux | Flux should always be capitalized.
+On-premises | On-premises or On-prem rather than On-premise or other variations.
+Multi-tenancy | Multi-tenancy or Multi-tenant rather than Multitenancy or other variations.
+
+## Shortcodes
+
```

Comment:
Maybe refer to https://fluxcd.io/docs/contributing/docs/writing-docs/ somewhere as well?

Reply:
I've mentioned at the top notes, the custom shortcodes for embedding content

Reply:
:+1: